api/authenticator.py

```diff
@@ -2174,6 +2174,21 @@ def _authentication_flow_document(self, _db):
         OPDS document.
         """
 
+        basic_doc = self._generate_authentication_flow_document(_db, type="http://opds-spec.org/auth/basic")
+        oauth_doc = self._generate_authentication_flow_document(_db, type="http://librarysimplified.org/authtype/OAuth-Client-Credentials")
```

Comment:
Per your other comment, yes, both of these should be constants in this class. Just not FLOW_TYPE, which you're right to have stopped using.